hkmc2/shared/src/main/scala/hkmc2/semantics/Symbol.scala

```diff
@@ -122,6 +122,9 @@ end BlockMemberSymbol
 sealed abstract class MemberSymbol[Defn <: Definition](using State) extends Symbol:
   def nme: Str
   var defn: Opt[Defn] = N
+  lazy val isGetter: Bool = defn match
+    case S(TermDefinition(_, Fun, _, Nil, _, S(_), _, _)) => true
+    case _ => false
```

Comment:
Maybe I misunderstood your suggestion. Did you suggest we pass another parameter to `MemberSymbol` to indicate if this symbol is a getter, instead of checking `defn` every time?

Reply:
Yes except not in `MemberSymbol`, where it doesn't belong, more like in `TermSymbol`.